src/editor/components/monaco-tools/fetch-line-suggestion.js

```diff
@@ -0,0 +1,73 @@
+import * as monaco from "monaco-editor/esm/vs/editor/editor.api";
+
+import { FETCH_CHUNK_TYPES } from "../../state-schemas/state-schema";
+import { makeSuggestionList } from "./make-suggestion-list";
+
+const { Keyword, Snippet, File } = monaco.languages.CompletionItemKind;
+
+export const fetchLineSuggestion = (lineSoFar, fileNames) => {
```

Comment:
Likewise, a test for this function sounds like a good idea, at least covering a few basic cases.

Reply:
added tests! :-)